Worker.Extension.Sql/src/Microsoft.Azure.Functions.Worker.Extension.Sql.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<AssemblyName>Microsoft.Azure.Functions.Worker.Extension.Sql</AssemblyName>
+		<RootNamespace>Microsoft.Azure.Functions.Worker.Extension.Sql</RootNamespace>
+		<Description>Sql extension for .NET isolated functions</Description>
+
+		<!--Version information-->
+		<VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
If you're going to put a version in here it should be a dev-only one (such as 99.99.99 like https://github.com/Azure/azure-functions-sql-extension/blob/main/src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj)